include/otdoa_al/otdoa_otdoa2al_api.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2025 PHY Wireless, Inc.
+ *
+ * SPDX-License-Identifier: LicenseRef-PHYW
+ */
+
+/**
+ * @note API is designed to be accessed by the OTDOA library.  In general,
+ * it should not be used by the user application code.
+ */
+
+#ifndef PHYWI_OTDOA2AL_API__
+#define PHYWI_OTDOA2AL_API__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "otdoa_nordic_at.h"
+#include "otdoa_api.h"
+
+int otdoa_start(void);
+int otdoa_stop(void);
+
+/**
+ * @brief Initialize the OTDOA AL library
+ * @param[in] event_callback Callback function used by the library
+ *                     to return results and status to the client
+ * @return 0 on success
+ */
+int32_t otdoa_al_init(otdoa_api_callback_t event_callback);
+
+/***************************************************************************
+ * Message Handling Functions
+ */
+int32_t otdoa_send_to_main_thread(const void *pvMsg, uint32_t u32Length);
+int otdoa_queue_rs_message(const void *msg, const size_t length);
```

Comment:
generally functions have documentation including what the variables are, what it does, when the return values mean